.eslintrc.js

```diff
@@ -18,6 +18,6 @@ module.exports = {
     'no-useless-catch': 'off',
     'no-useless-escape': 'off',
     'valid-typeof': 'off',
-    '@typescript-eslint/no-unused-vars': 'error',
+    '@typescript-eslint/no-unused-vars': [1, { args: 'after-used' }],
```

Comment:
I'd prefer `args: none` for now.
Also, please use 'warning' instead of a magic constant.

Reply:
Change on 462bf9f